lms/djangoapps/verify_student/models.py

```diff
@@ -141,6 +141,34 @@ def active_at_datetime(self, deadline):
         )
 
 
+class ManualVerification(IDVerificationAttempt):
+    """
+    Each ManualVerification represents an enterprise user's verification that bypasses the need for
+    any other verification.
+    """
+
+    reason = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text=(
+            'Specifies the reason for manual verification of the user.'
+        )
+    )
+
+    class Meta(object):
+        app_label = "verify_student"
```

Comment:
nit: stick with single quotes

Reply:
double quotes are being used by all the other models in the file ...